Jenkinsfile

```diff
@@ -69,32 +69,19 @@ node('vetsgov-general-purpose') {
     if (commonStages.shouldBail() || !commonStages.VAGOV_BUILDTYPES.contains('vagovprod')) { return }
     dir("vets-website") {
       try {
-        if (commonStages.IS_PROD_BRANCH && commonStages.VAGOV_BUILDTYPES.contains('vagovprod')) {
```

Comment:
I removed the `if...else` in 850bc842366d9fad2e0d8b7bddbafce42905e346 because both blocks were identical after adding `nightwatch-accessibility` to the `else` block. 

If removing the `if...else` makes this change too risky, I don't mind reverting 850bc842366d9fad2e0d8b7bddbafce42905e346 to get this PR back to the state it was in at d62ec3cbc69a13c7cf5dc8e6c2462e244a5e9776.